src/Tribe/Admin/Notice/Php_Version.php

```diff
@@ -8,7 +8,7 @@ public function hook() {
 
 		// display the PHP version notice
 		tribe_notice(
-			'php-deprecated',
+			'php-deprecated-74',
```

Comment:
This doesn't need to change, but I'm not sure changing this each time is wise 🤔 
(i.e. are we going to change this to `php-deprecated-82` at some point?)

Reply:
Also thinking out loud - does it make sense to make the required version a (string) param, then it only needs to get changed in one place 🤔